hy/compiler.py

```diff
@@ -279,32 +281,54 @@ def is_unpack(kind, x):
             and x[0] == "unpack-" + kind)
 
 
-_stdlib = {}
+class HyASTCompiler(object):
+    """A Hy-to-Python AST compiler.
 
+    It's probably more precise to call this a transpiler, though.
```

Comment:
Transpliers are compilers though. And Python AST is a lower-level representation rather than human-readable source code, so "transpiler" isn't really accurate.

Reply:
Ha, yeah, I was debating that in my head, and ultimately sided with "transpiler".  Still, that comment was actually a personal note that shouldn't have ended up in there!